### PR TITLE
OW-938 - Add new alpha build to gh pages

### PR DESCRIPTION
**Pull request by maikthomas**
- Add new alpha build to versions
- update link to s3 with link to gh pages
- remove references to s3 for assets and use local media/ path
- fix require of module we are not using (and is not in deps so breaks build command)
- Add check for package.json version to make sure it's not using `^`, `~` or `>`

### CODE REVIEW

File: ML-Transformers/BackgroundEnchantments/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Background Effects Sample Application
 This sample application shows the result of applying some background visual effects by using the [@vonage/ml-transformers](https://www.npmjs.com/package/@vonage/ml-transformers) library.
 # Demo
-You can see a live demo of this sample running [here](https://vonage-background-enchantments-sample.s3.amazonaws.com/index.html)
+You can see a live demo of this sample running [here](https://vonage.github.io/vonage-media-transformers-samples/)
 # Building the application
```
Comment (maikthomas) [inline ML-Transformers/BackgroundEnchantments/README.md:5]: Replaced this link to the s3 with the link to github pages

File: ML-Transformers/BackgroundEnchantments/copyToDocs.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-const ejs = require('ejs');
```
Comment (maikthomas) [inline ML-Transformers/BackgroundEnchantments/copyToDocs.js:1]: this snuck in by accident last PR

File: ML-Transformers/BackgroundEnchantments/README.md (hunk 1 above)
Comment (joliveraortega) [reply ML-Transformers/BackgroundEnchantments/README.md:5]: Good catch!

File: ML-Transformers/BackgroundEnchantments/main.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import { setVonageMetadata } from "@vonage/media-processor";
 import packageInfo from './package.json';
 
 const MEDIA_ASSETS_URI: string =
-  "https://vonage-background-enchantments-sample.s3.amazonaws.com/";
+  "media/";
```
Comment (maikthomas) [inline ML-Transformers/BackgroundEnchantments/main.ts:16]: We were loading the assets used for video and image background from s3 too so changed that to local media/ route

**Review by joliveraortega — APPROVED** [review]
LGMT, thanks!

**Pull request merged**